lib/plugins/accept.js

```diff
@@ -38,7 +38,7 @@ function acceptParser(accepts) {
             return a;
         })
         .map(function map(a) {
-            return a.indexOf('/') === -1 ? mime.lookup(a) : a;
+            return a.indexOf('/') === -1 ? mime.getType(a) : a;
```

Comment:
What is the difference between `mime.lookup` and `mime.getType`?

Reply:
`lookup` was renamed to `getType` in the newer version of `mime`.